test/ads/vast/VASTClient.spec.js

```diff
@@ -132,7 +132,7 @@ describe("VASTClient", function () {
         sinon.assert.calledWith(vast._requestVASTXml, 'http://fake.url');
         flushVASTXmlRequest(null, vastXML('<Ad><Wrapper><VASTAdTagURI><![CDATA[http://VASTAdTagURI.com]]></VASTAdTagURI></Wrapper></Ad>'));
 
-        assertError(callback, 'on VASTClient.getVASTAd.getAd, players wrapper limit reached (the limit is 1)', 302);
+        assertError(callback, 'on VASTClient.getVASTAd.getAd, player's wrapper limit reached (the limit is 1)', 302);
```

Comment:
you need to escape the single quote here

Reply:
I believe the bug you pointed to was corrected in my 2nd commit to this PR, @vpitts. Please verify, or let me know if I've overlooked something else. Thanks!

Reply:
I see the fix. 

However, I notice that is some place the string is wrapped in double quotes (which removes the requirement for the escape character) and in other places the string is wrapped in single quotes.  Wouldn't it better to do it the same in each place.  Eugene mentioned that LINT prefers the use of single quotes.  I just seems like less confusion if we do it the same in all places.